src/sbus/server/sbus_server.c

```diff
@@ -146,6 +146,10 @@ sbus_server_symlink_create(const char *filename,
 {
     errno_t ret;
 
+    if (symlink_filename == NULL) {
+        return ENOMEM;
+    }
+
```

Comment:
> There are no checks that talloc()-ed symlink file name is not NULL.

Strictly speaking, there is a check:
https://github.com/SSSD/sssd/blob/273ff9e3ea2fa2838458b47950420f67bc833764/src/sbus/server/sbus_server.c#L99

But having additional check here won't hurt. Though IMO it would be more suitable to use 'EINVAL'  here.